docsbuild/content/migrations/role.md

```diff
@@ -35,6 +35,41 @@ Add a role to keycloak, fails if the role already exists
           role:
           - value1
           - value2
+
+## updateRole
+Update an existing realm- or client-role in keycloak.
+### Parameter
+- realm: String, optional
+- name: String, not optional,
+- clientId: String, optional, default=realmRole,
+- description: String, optional, default=""
```

Comment:
The default parameter are not quite useful in this case. If you don't specify an optional parameter of the `updateRole`-Action it will just leave this field as it is. On behalf of the `description` parameter see below please :)

Reply:
copy paste foo, adjusted :) I also renamed `Parameter` to `Parameters`. I'll ship another MR where i adjust it for al